Update README.md (#1)
eleanorjboyd/grocery-data-processor: README.md

## Getting started

> 1. Create a venv using the VS Code "Python: Create Environments" command in the command palette and select to install the required dependencies from `requirements.txt`

Author: ```suggestion
1. Create a venv using the VS Code **Python: Create Environment** command found in the Command Palette (`Ctrl/Cmd + P`) and select to install the required dependencies from `requirements.txt`
```

Here are the following configurable options:
```

> 

Author: ```suggestion
```

```
"python.configs": [
        {
         // Testing config example 

>             "name": "config unittest 1", // required and must be unique

Author: ```suggestion
            "name": "basic config 1", // required and must be unique
```
